pki/services.go

```diff
@@ -555,6 +577,13 @@ func GenerateRKEServicesCerts(ctx context.Context, certs map[string]CertificateP
 	}
 	if IsKubeletGenerateServingCertificateEnabledinConfig(&rkeConfig) {
 		RKECerts = append(RKECerts, GenerateKubeletCertificate)
+	} else {
+		//Clean up kubelet certs which generated when GenerateServingCertificate enabled
+		for k := range certs {
```

Comment:
Why do we remove all kubelet certs here

Reply:
When we rke up, the generate serving certificate is disabled in default,and there haven’t any kubelet cert in cluster.rkestate:
￼
![image](https://user-images.githubusercontent.com/37337250/71069539-e864bc80-21b3-11ea-99c3-2576bc9260e1.png)

And if we   enable  generate serving certificate and rke up again,it will generate kubelet certs,we can see them in cluster.rkestate:
![image](https://user-images.githubusercontent.com/37337250/71069547-f0bcf780-21b3-11ea-85ca-91959ebb96fb.png)

￼
But if we disable generate serving certificate  and rke up again, the kubelet certs will be unused and left in cluster.rkestate:
![image](https://user-images.githubusercontent.com/37337250/71069568-fadef600-21b3-11ea-8323-4e15605ef069.png)

￼
So we need to delete all unused kubelet certs when  generate serving certificate is disabled ~